pkg/process/checks/process.go

```diff
@@ -41,8 +41,9 @@ const (
 func NewProcessCheck(config ddconfig.ConfigReader) *ProcessCheck {
 	var extractors []metadata.Extractor
 	var wlmServer *workloadmeta.GRPCServer
+	var wlmExtractor *workloadmeta.WorkloadMetaExtractor
 	if workloadmeta.Enabled(config) {
-		wlmExtractor := workloadmeta.NewWorkloadMetaExtractor(config)
+		wlmExtractor = workloadmeta.NewWorkloadMetaExtractor(config)
```

Comment:
Optional: I think it might be cleaner to encapsulate all of the workloadmeta variables into this block, like so:

```go
check := &ProcessCheck{
	... instantiate variables as normal ...
}

if workloadmeta.Enabled() {
	check.workloadMetaExtractor = workloadmeta.NewExtractor()
	... etc etc ...
}

return check
```

Reply:
Great suggestion!!! This makes the code much cleaner. I went ahead and applied this change 😄